testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapplist/MiniAppListFragment.kt

```diff
@@ -63,6 +71,7 @@ class MiniAppListFragment : BaseFragment(), MiniAppList {
             miniAppListData.observe(viewLifecycleOwner, Observer {
                 swipeRefreshLayout.isRefreshing = false
                 displayMiniAppList(it)
+                downloadedList = it
```

Comment:
Do you think this assignation can happen directly in `displayMiniAppList`; which could be refactored to updateMiniAppListState as you are also managing the state of emptyView over there (at minimum)?

Reply:
I agree that we could assign it in somewhere else, also we could have a `updateMiniAppListState` functionality. There are two use-cases e.g. adding and updating list, I have updated this part by https://github.com/rakutentech/android-miniapp/pull/111/commits/84bf23969769b0ab550de1bd907144ef2418f5cd.